composer.json

```diff
@@ -28,6 +28,7 @@
     "require-dev": {
         "digitalrevolution/phpunit-file-coverage-inspection": "^v2.0.0",
         "mikey179/vfsstream": "^1.6.11",
+        "nikic/php-parser": "^4.10",
```

Comment:
nikic/php-parser 5+ crashes code coverage. Hard to reproduce without running code coverage.